src/stkSWIV.sol

```diff
@@ -607,6 +606,18 @@ contract stkSWIV is ERC20 {
         return (sharesRedeemed, bptOut, [amountsOut[0], amountsOut[1]]);
     }
 
+    //////////////////// FEE DONATION ////////////////////
+
+    // Method to donate a BPT amount to the SSM
+    // @param: amount - amount of BPT to donate
+    // @returns: the amount of BPT donated
+    function donate(uint256 amount) public {
+        // Transfer the BPT to the SSM
+        SafeTransferLib.transferFrom(balancerLPT, msg.sender, address(this), amount);
+        // Emit donation event
+        emit Donation(amount, msg.sender);
```

Comment:
one question here... how should this affect rewards calculation on the BE? it shouldn't imo 🤷‍♂️